docs/proposal/20250314-optional-controllers.md

```diff
@@ -0,0 +1,73 @@
+# Problem
+
```

Comment:
Might be nice to explicitly state this supersedes the other proposal in the front matter.